Python/defragment_data_of_existing_hyper_file.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+from pathlib import PurePath
+from tableauhyperapi import HyperProcess, Telemetry, Connection, TableDefinition, TableName
+
```

Comment:
Can we please add a general comment to this script what it does?

Reply:
Added: 
 An example of how you can optimize the file storage of an existing `.hyper` file by copying 
 all of the tables and data into a new file. This reduces file fragmentation.